modules/ballerina-http/src/main/ballerina/ballerina/net/http/resiliency/circuit-breaker.bal

```diff
@@ -0,0 +1,226 @@
+package ballerina.net.http.resiliency;
+
+import ballerina.net.http;
+
+enum CircuitState {
+    OPEN, CLOSE, HALF_OPEN
+}
+
+struct CircuitHealth {
+    float requestCount;
```

Comment:
Why is it float but not int ?

Reply:
So that there is no need to cast to float when calculating the current failure ratio in line 193